README.md

```diff
@@ -324,6 +325,46 @@ You can automatically reformat your commit via:
 $ cargo fmt --all
 ```
 
+## Non-Consensus Breaking Release Process
+
+For non-consensus breaking releases, this project uses the following release process:
+
+1. A release manager enumerates the PRs or issues that would _block_
+   the release. A label should be applied to each such issue/PR as
+   `2.1.x.y.z-blocker`. The release manager should ping these
+   issue/PR owners for updates on whether or not those issues/PRs have
+   any blockers or are waiting on feedback.
+1. Once all `2.1.x.y.z-blocker` PRs have been merged to `develop`, the release
+   manager will open a `develop -> master` PR.
+1. Immediately following this, the release manager will then create a new tag
+   by manually triggering the [`stacks-blockchain` Github Actions workflow](https://github.com/blockstack/stacks-blockchain/actions/workflows/stacks-blockchain.yml)
```

Comment:
is this by clicking "Run Workflow"? Maybe you can add some text or even optionally a picture.

Reply:
Added more specificity in 2279149